updater/Gemfile.lock

```diff
@@ -521,7 +522,7 @@ CHECKSUMS
   aws-sdk-core (3.190.3) sha256=f9b66775d79112bcffcd031aa2c97aac2b1819fa2d917b1c72557635260f3d41
   aws-sdk-ecr (1.68.0) sha256=86763b7291170dc6dbbf827f35d4142683622e68d7dff6719eeb98b13b677e80
   aws-sigv4 (1.8.0) sha256=84dd99768b91b93b63d1d8e53ee837cfd06ab402812772a7899a78f9f9117cbc
-  base64 (0.1.1) sha256=0c75d351a429b5176a476cd8a3740cff3277d2bac26a50b5c7456c266e9acd33
+  base64 (0.2.0) sha256=0f25e9b21a02a0cc0cea8ef92b2041035d39350946e8789c562b2d1a3da01507
```

Comment:
🔎 Noting that a major version bump is occurring here

Reply:
The major version bump dropped support for Ruby 2.3. See https://github.com/ruby/base64/releases/tag/v0.2.0